setup.py

```diff
@@ -80,6 +81,8 @@
             "black==20.8b1",
             "grpcio-tools==1.32.0",
             "isort==5.7.0",
+            "pre-commit==2.16.0",
+            "pylint==2.12.2",
```

Comment:
Is there some description here of what we would need to do to get `pre-commit` set up?

Reply:
user just has to run `pre-commit install`, where should i include this instruction?

Reply:
I think add instructions for setup in in the `CONTRIBUTING.md`.

Reply:
You will need to take a read-through and update the file.